tests/acceptance/test_build.py

```diff
@@ -739,4 +739,10 @@ def test_build_artifact_depends_and_provides(
         output = run_verbose("mender-artifact read %s" % image, capture=True).decode()
         other = TestBuild.BuildDependsProvides.parse(output)
 
+        # MEN-2956: Mender-Artifact now writes rootfs-image-checksum by default.
+        # Hence, the checksum should be present in all these test cases.
+        assert "rootfs_image_checksum" in other.provides, "Empty rootfs_image_checksum in the built rootfs-image artifact, this should be added by default by `mender-artifact write rootfs-image`"
```

Comment:
You will need to bump the `min_mender_version` above to `2.3.0` (should be fine to not test 2.2.0, since depends/provides are not used for anything important there).